### DISCUSSION TITLE
3kh0 the v108 method doesn't work because inspect isn't allowed on my Chromebook

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#331] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
inspect isn't available for me and btw is this just temporary until pingot comes out

**Comment by Aka-but-nice** [comment, marked as answer]
THATS WHAT IM SAYING, like at that point if you got inspect you literally have access to chrome api like

**Reply by Theblackmamba824** [reply]
fax

**Reply by Aka-but-nice** [reply]
not me

**Comment by Sxri** [comment]
do bookmarklets work? if so, you could try using firebug, then use the cloned version.

**Reply by Husk-ees** [reply]
Bypassi's blog explains how to exploit this, just do open() in console and run the code
